yb-voyager/cmd/live_migration.go

```diff
@@ -88,10 +116,60 @@ func handleEvent(event *tgtdb.Event) error {
 	if err != nil {
 		return fmt.Errorf("error transforming event key fields: %v", err)
 	}
-	batch := []*tgtdb.Event{event}
-	err = tdb.ExecuteBatch(batch)
-	if err != nil {
-		return fmt.Errorf("error executing batch: %v", err)
-	}
+	insertIntoTargetEventChan(event, targetEventChans)
 	return nil
 }
+
+func insertIntoTargetEventChan(e *tgtdb.Event, targetEventChans []chan *tgtdb.Event) {
+	// hash event
+	delimiter := "-"
+	stringToBeHashed := e.SchemaName + delimiter + e.TableName
```

Comment:
Nice!. In this case though, I am going to use multiple hash.Writes (instead of concating a string) which is essentially just writing byte buffers, so I think this should be convered